themes/digital.gov/src/scss/new/shortcodes/_highlight.scss

```diff
@@ -1,7 +1,8 @@
 @use "uswds-core" as *;
 
 .highlight {
-  @include u-padding-y("1px");
-  @include u-padding-x("05");
-  @include u-bg("yellow-10v");
+  code {
+    background: none;
+    border: none;
+  }
```

Comment:
I'm not sure why we are removing this highlight? Now with the short code around text, there is no effect. Should we remove the short code all together? 

<img width="746" alt="Screenshot 2024-04-25 at 9 34 16 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/cb18e70a-b391-4e88-9fb6-2f72b1eaa99f">

Reply:
updated the code so that the yellow highlight was added back when needed